_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -1,17 +1,190 @@
+{: .notice--info}
 
-- This could be structured Attributes which describe someone, e.g. names, addresses, or communication channels so-called IdentityAttributes.
-- It could be Attributes of structured or unstructured certifications of this someone, e.g. school certificates, bachelor degrees or language certifications, e.g. for IdentityAttributes which are tagged with a semantic meaning.
-- It could be peer specific relationship or contractual details, so-called RelationshipAttributes, e.g. student ids, contract details or relationship-specific settings
-- Or it could be data of other relationships (i.e. RelationshipAttributes), e.g. a bonus program number of a third party.
+## Request for reading Attributes
+
+The Sender wants to read an Attribute of the Recipient. To do this, the Sender must first create a suitable [Request]({% link _docs_integrate/data-model-overview.md %}#request), which it can then send to the Recipient. In the following subsections, we describe the general appearance of a Request for reading Attributes.
+
+### Role of ReadAttributeRequestItem
+
+For reading a single Attribute, the Sender needs to insert a single RequestItem of type [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) into the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request). The input it has to provide for the `query` property of the [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) depends on what type of Attribute it wants to get. If the Sender wants to read an IdentityAttribute, it must use an appropriate [IdentityAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#identityattributequery). If it is interested in a RelationshipAttribute of the Recipient, it must insert a [RelationshipAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#relationshipattributequery) or a [ThirdPartyRelationshipAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#thirdpartyrelationshipattributequery) at this point. This depends on whether the Sender wants to read a RelationshipAttribute that the Recipient has in the context of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with the Sender or with a third party.
+
+### Combinations and usage scenarios of ReadAttributeRequestItem
+
+The following table provides an overview of the possible kinds of Attributes that the Sender can read from the Recipient using the ReadAttributeRequestItem. It must be taken into account whether the [Attribute]({% link _docs_integrate/data-model-overview.md %}#attributes) is an IdentityAttribute or a RelationshipAttribute and which Identity is its `owner`. If the Sender wants to read a RelationshipAttribute from the Recipient, a distinction must be made between which Identities the Relationship in question exists.
```

Comment:
I have linked it again because I am referring to a property of the data objects at this point. I'm afraid that the reader might be lost if you use a link to a data object only once at the beginning of a long documentation page, if properties of a data object are mentioned sometime later. Anyway, I'm trying to reduce the number of repetitive links and find a compromise between avoiding redundancy, clarity for the reader and consistency with the other guides.

I have now removed a few more links and will keep an eye on this. Perhaps someone else can also give their opinion on this.

Reply:
I would say that every or at least always the first occurrence in a paragraph, should be linked. I don't think it disturbs the flow of reading, it's just a colored word

Reply:
eg. https://www.php.net/manual/en/language.generators.overview.php also links each occurrence (the best programming language :P)